packages/web/src/components/user-badges/FollowsYouBadge.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react'
+import styles from './FollowsYouBadge.module.css'
+
+export default function FollowsYouBadge() {
+  return <div className={styles.badge}>FOLLOWS YOU</div>
+}
```

Comment:
Make sure you have Prettier installed, and if you're using vscode you can set format on save to true!